site/faq/README.md

```diff
@@ -4,59 +4,45 @@
 
 Please visit the [roadmap document] and the [kpt milestones].
 
-### What does kpt provide that git clone doesn't?
-
-kpt enables out-of-the-box workflows that `git clone` does not. Such as: cloning
-and versioning git subdirectories, updating from upstream by performing
-structured merges on resources, programmatically editing configuration (rather
-than with an editor), etc.
-
 ### How is kpt different from other solutions?
 
-Rather than expressing configuration as code, kpt represents configuration
-packages as data, in particular as YAML or JSON objects adhering to [The
-Kubernetes Resource Model].
+Rather than expressing configuration as code that generates configuration,
+kpt represents [Configuration as Data], in particular as YAML or JSON objects
+adhering to [The Kubernetes Resource Model], and uses a transformation-based
+approach to customization. This enables interoperability of a variety of generators,
+transformers, and validators. One doesn’t have to make all changes through a monolithic
+generator implementation.
 
 ### What's the difference between kpt and kustomize?
 
-[Kustomize] is a CNCF project that is a part of Kubernetes. While both kpt and
-kustomize support customization of KRM manifests without use of templates or
-DSL(s), there are important differences in both feature set and the scope of
-these projects. kpt supports end-to-end solutions for packaging, customization
-and actuation of resources. In kustomize, packaging is explicitly out of scope
-and actuation is deferred to kubectl.
+While both kpt and kustomize support customization of KRM resources via a transformation-based approach,
+there are important differences in both feature sets and the scopes of these projects. 
 
-Here are the strengths and investment areas for these products:
+_kustomize_
+
+- Builds the final configuration out of place, primarily using the [overlay pattern].
+- Treats base layers as immutable, but enables nearly arbitrary overrides.
 
 _kpt_
 
-- Supports both in-place and out-of-place customization. kustomize is only
-  focused on providing out-of-place hydration using the [overlay pattern].
-- Allows you to edit the configuration in-place without creating complex patches.
-- Focuses on rebase with resource merge strategy allowing for edited config to
+- Optimizes for WYSIWYG configuration and in-place customization.
+- Allows edits to the configuration in-place without creating complex patches.
+- Supports rebase with resource merge strategy allowing for edited config to
   be updated.
-- Has a continuous learning curve as you usually start small with modifying
-  several YAML files using an editor and then want to scale with complexity of
-  the application.
 - Enables workflows that combine programmatic changes (functions) with manual
```

Comment:
nit: square brackets around `functions`

Reply:
Thanks. Looks like that was a preexisting bug. Should be fixed now.